portfolio-project-matching-app/models/Project.js

```diff
@@ -73,6 +73,19 @@ class Project {
         return project;
     }
 
+    hasSearchKey(key) {
+        /*
+        DESCRIPTION:    indicates whether search key is present in Project
+                        object's name property
+
+        INPUT:          key (string): search key applied to name property
+
+        RETURN:         boolean indication as to whether the search key appears
+                        in the Project's name
+        */
+        return this.name.includes(key);
+    }
+
```

Comment:
Really only created this to improve readability in `browseProjects`. Hoping the name is self-descriptive enough that the reader does not need to interpret code to find out what I'm doing.